spec/features/food_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Test user', email: 'test444@gmail.com', password: '123456',
+                        password_confirmation: '123456', confirmation_token: nil, confirmed_at: Time.now)
+    @recipe = Recipe.create(name: 'Test recipe', preparation_time: 10.2, cooking_time: 20.3,
+                            description: 'Test description', public: true, user_id: @user.id)
+    @food = Food.create(name: 'Test food', price: 12.2, quantity: 4, measurement_unit: 'gram', user_id: @user.id)
+    @food_two = Food.create(name: 'Test food Two', price: 12.2, quantity: 4, measurement_unit: 'gram',
+                            user_id: @user.id)
+    @recipe_food = RecipeFood.create(quantity: 10, recipe_id: @recipe.id, food_id: @food.id)
+    visit new_user_session_path
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: @user.password
+    click_button 'Log in'
+  end
```

Comment:
Hi @Lancelot-SO if this is integration test, you have to create assertions to check if web page displays everything as expected. You have to write tests for Food items index page, food item new page etc. You also have to write test cases for Show public recipes index page

Reply:
I am creating for the other model files too. i just pushed it because I want to do it one after the other

Reply:
I have added test for the other models hope you check them and review them.